<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAsideUp</class>
 <widget class="QDialog" name="DialogAsideUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Напоминания</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableView" name="mpAsides">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mpAsideDescription" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>270</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>отложить до</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCalendarWidget" name="mpCalendar">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>220</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="mpAside">
          <property name="text">
           <string>Отложить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="mpEditor">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>245</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
     <zorder></zorder>
     <zorder>mpEditor</zorder>
     <zorder>mpEditor</zorder>
     <zorder></zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
